packages/athena/tsconfig.json

```diff
@@ -1,8 +1,17 @@
 {
   "extends": "../../tsconfig-base.json",
   "compilerOptions": {
+    "noEmitOnError": false,
```

Comment:
Because it blocks rebuilds during dev which makes debugging things really hard since you have to fix all type-related issues just to see if your change even works